code/src/components/Overview.js

```diff
@@ -0,0 +1,48 @@
+import React from 'react';
+import styled from 'styled-components/macro';
+import { Wrapper, Actionbtn } from 'styles/GlobalStyles';
+import { tasks } from 'reducers/tasks';
+
+export const Overview = ({ incomplete, allTasks, dispatch }) => {
```

Comment:
I see that you pass some props here from Tasklist.js. It looks like you're using the "incomplete" data only in this component. Did you think about getting it from the store here instead?